src/models/asyncapi.ts

```diff
@@ -31,4 +38,15 @@ export interface AsyncAPIDocumentInterface extends BaseModel<v2.AsyncAPIObject |
   allOperations(): OperationsInterface;
   allMessages(): MessagesInterface;
   allSchemas(): SchemasInterface;
+  allSecuritySchemes(): SecuritySchemesInterface;
+  allServerVariables(): ServerVariablesInterface;
+  allParameters(): ChannelParametersInterface;
+  allCorrelationIds(): CorrelationIdsInterface;
+  allTags(): TagsInterface;
+  allOperationTraits(): OperationTraitsInterface;
+  allMessageTraits(): MessageTraitsInterface;
+  allServerBindings(): BindingsInterface;
```

Comment:
If it doesn't have a clear use case, please drop it. Same with any other method if it is the case.

Context: https://github.com/asyncapi/parser-api/issues/109#issuecomment-2015374655